public/js/controllers/index.js

```diff
@@ -57,8 +61,38 @@ angular.module('mean.system')
         };
 
         $scope.playAsGuest = () => {
-          game.joinGame();
-          $location.path('/app');
+          if ($scope.region === undefined || $scope.region === '') {
+            $scope.error = 'Select a region first!';
+          } else {
+            $('#close').click();
+            // $location.path('/app');
+          }
+        };
+
+        $scope.playWithStrangers = () => {
+          // game.joinGame();      
+          if ($scope.region === undefined || $scope.region === '') {
+            $scope.error = 'Select a region first!';
+          } else {
+            $location.path('/app');
+            $('#close').click();
+          }
+        };
+
+        $scope.playWithFriends = () => {
+          if ($scope.region === undefined || $scope.region === '') {
+            $location.search('error=selectRegion');
+            return;
+          }
+          localStorage.setItem('region', $scope.region);
+          $scope.hideModal();
+          $location.path('/play?custom');
+          $window.location.href = '/play?custom';
```

Comment:
Not sure if this is needed here, line above seems to be do same

Reply:
All done, thanks